tensorflow/python/keras/benchmarks/keras_examples_benchmark_test.py

```diff
@@ -40,38 +39,28 @@ class KerasExamplesBenchmark(
       Other details can see in `measure_performance()` method of
       benchmark_util.
   """
-  """The parameters of each benchmark is a tuple:
-
-     (benchmark_name_suffix, batch_size, run_iters).
-     benchmark_name_suffix: The suffix of the benchmark test name with
-     convention `{bs}_{batch_size}`.
-     batch_size: Integer. Number of samples per gradient update.
-     run_iters: Integer. Number of iterations to run the
-         performance measurement.
-  """
-  _benchmark_parameters = [('bs_32', 32, 2), ('bs_64', 64, 2),
-                           ('bs_128', 128, 1), ('bs_256', 256, 1),
-                           ('bs_512', 512, 3)]
 
   def _lstm_imdb_model(self):
     """LSTM model from https://keras.io/examples/nlp/bidirectional_lstm_imdb/."""
     inputs = tf.keras.Input(shape=(None,), dtype='int32')
     x = tf.keras.layers.Embedding(_MAX_FEATURE, 128)(inputs)
     x = tf.keras.layers.Bidirectional(
-        tf.keras.layers.LSTM(64, return_sequences=True))(
-            x)
+        tf.keras.layers.LSTM(
+            64, return_sequences=True))(x)
     x = tf.keras.layers.Bidirectional(tf.keras.layers.LSTM(64))(x)
     outputs = tf.keras.layers.Dense(1, activation='sigmoid')(x)
     model = tf.keras.Model(inputs, outputs)
     return model
 
-  def benchmark_bidirect_lstm_imdb(self, batch_size, run_iters):
+  def benchmark_bidirect_lstm_imdb(self):
```

Comment:
All parameterized benchmarks should be converted to `benchmark_xxx` test. So you should have 5 benchmark methods here.

Reply:
> All parameterized benchmarks should be converted to `benchmark_xxx` test. So you should have 5 benchmark methods here.

Do you mean I need to add 5 benchmarks for different models from keras.io after this PR?

Reply:
In theory, for equivalent conversion, it might be good to have 5 benchmark test.

I think it also depends on how large the dataset is in the benchmark. If the dataset size is large, you may use larger batch size for it to make the benchmark run faster. So we need to know how many examples in the dataset before determining the `batch_size` number.

For this one, 1 benchmark test is not enough. If 5 is too many, let's keep it as 3?

Reply:
So, we can keep the similar configuration like `(batch_size = 32, run_iters=2)`, `(batch_size =64, run_iters=3)`, `(batch_size =64, run_iters=1)`, is it correct? If so, I'd like rename as this format `benchmark_model_bs_xx_iter_xx`. It's more straightforward to us.

Reply:
Maybe `benchmark_model_bs_xx`? `iter` can be ignored in the name, as it's more about how to collect benchmark results, not how the models are tested. 

Later if dist-strat is applied, # of gpus can be added to the name.

Reply:
Sounds great to me.  I will update this PR tomorrow.